front/app/containers/Admin/invitations/all/index.tsx

```diff
@@ -55,185 +53,103 @@ const EmptyStateContainer = styled.div`
-                    />
-                  </Table.HeaderCell>
-                </Table.Row>
-              </Table.Header>
-
-              <Table.Body>
-                {invitesList.map((invite) => (
-                  <Row key={invite.id} invite={invite} />
-                ))}
-              </Table.Body>
-
-              {currentPage && lastPage && lastPage > 1 && (
-                <Table.Footer fullWidth={true}>
-                  <Table.Row>
-                    <Table.HeaderCell colSpan="6">
-                      <Pagination
-                        currentPage={currentPage}
-                        totalPages={lastPage}
-                        loadPage={onChangePage}
-                      />
-                    </Table.HeaderCell>
-                  </Table.Row>
-                </Table.Footer>
-              )}
-            </Table>
-          )}
-
-          {isEmpty(invitesList) && !isEmpty(searchValue) && (
-            <EmptyStateContainer>
-              <FormattedMessage
-                {...messages.currentlyNoInvitesThatMatchSearch}
-              />
-            </EmptyStateContainer>
+  return (
+    <Container>
+      <HeaderContainer>
+        <SearchInput
+          onChange={handleChangeSearchTerm}
+          a11y_numberOfSearchResults={invitesList.length}
+        />
+        <Button
+          buttonStyle="cl-blue"
+          icon="download"
+          onClick={handleInvitesExport}
+          processing={exporting}
+        >
+          <FormattedMessage {...messages.exportInvites} />
+        </Button>
+      </HeaderContainer>
+
+      {invitesList.length > 0 && (
+        <Table>
+          <TableHeader
+            sortAttribute={sortAttribute}
+            sortDirection={sortDirection}
+            onSortHeaderClick={handleSortHeaderClick}
+          />
+
+          <Body horizontalBorders>
+            {invitesList.map((invite) => (
+              <TableRow key={invite.id} invite={invite} />
+            ))}
+          </Body>
+
+          {currentPage && lastPage && lastPage > 1 && (
+            <Footer>
+              <Row>
+                <HeaderCell colSpan="5">
+                  <Pagination
+                    currentPage={1}
+                    totalPages={5}
+                    loadPage={onChangePage}
+                  />
+                </HeaderCell>
+              </Row>
+            </Footer>
           )}
-        </Container>
-      );
-    }
-
-    return null;
-  }
-}
-
-export default (inputProps: InputProps) => (
-  <GetInvites>
-    {(invites) => <InvitesTable {...inputProps} {...invites} />}
-  </GetInvites>
+        </Table>
+      )}
+
+      {isEmpty(invitesList) && !isEmpty(searchValue) && (
+        <EmptyStateContainer>
+          <FormattedMessage {...messages.currentlyNoInvitesThatMatchSearch} />
+        </EmptyStateContainer>
+      )}
+    </Container>
+  );
+};
+
+export default () => (
+  <GetInvites>{(invites) => <InvitesTable {...invites} />}</GetInvites>
```

Comment:
Would be cool to replace this `GetInvites` hook as well but it's some pretty bad RxJS spaghetti so I skipped it for now